src/activities/Elsa.Activities.Timers/Options/TimersOptions.cs

```diff
@@ -1,11 +1,16 @@
+using Microsoft.Extensions.DependencyInjection;
+
 using NodaTime;
 
 namespace Elsa.Activities.Timers.Options
 {
     public class TimersOptions
     {
-        public TimersOptions()
+        public IServiceCollection Services { get; }
+
+        public TimersOptions(IServiceCollection services)
         {
+            Services = services;
             SweepInterval = Duration.FromMinutes(1);
```

Comment:
This reminds me: the `SweepInterval` isn't used anymore. It was used before I added Quartz as the background task scheduler.